doc/dev/sharp_bits.rst

```diff
@@ -1251,6 +1252,10 @@ Note that ``jax.vmap`` cannot be used within a qjit-compiled function:
 >>> qjit(jax.vmap(circuit))(jnp.array([0.1, 0.2]))
 NotImplementedError: Batching rule for 'qinst' not implemented
 
+In addition, shot-vectors are currently only supported in a limited manner;
+shot-vectors work with ``qml.sample``, but not other measurement processes
+such as ``qml.expval`` and ``qml.probs``.
```

Comment:
You can link to the documentation of these functions. See what I did [here](https://github.com/PennyLaneAI/catalyst/pull/1248/files). 

To test locally we have `make docs`. The built webpages are in `doc/build/index.html` (or some similar path name, I don't recall it off the top of my head but it's there)

Reply:
```suggestion
such as :func:`~.expval` and :func:`~.probs`.
```

Reply:
Using ``:func:`~.expval` `` didn't produce a link—I assume because within the Catalyst docs, `~` refers to the Catalyst namespace, not PennyLane. Using ``:func:`qml.expval <pennylane.expval>` `` does work, however.